### ISSUE TITLE
FNX2-13152 ⁃ [Bug][a11y] Can't interact using one finger on PiP if Talkback is enabled

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#10389]
### Prerequisites
The talkback option is enabled from the Android settings.

## Steps to reproduce
1. Open a video in youtube.com;
1. Go to full-screen;
1. Display the Picture-in-picture video mode.
1. Try to interact with the PiP screen using 1 or 2 fingers.

### Expected behavior
The user can expand the PiP screen with 1 finger.

### Actual behavior
The user can expand the PiP screen only with 2 fingers.

### Device information
Android device: Xiaomi Mi 8 Lite (Android 9).
Fenix version: Firefox Preview Nightly 4/30 #21210606.
Not reproducible using LG G7 Fit (Android 8.1).
[Video](https://drive.google.com/file/d/1W1nkU58GJ3cjOykpY45QwttCsZAz7_sZ/view?usp=sharing).

**Comment by mcarare** [comment]
I'm not sure about what the issue is here. I cannot reproduce this, and it is not obvious from the video in the issue what touches are made. As to my understanding, some kind of interaction is possible. Is the snackbar the problem? 
It would be helpful if the same issue would be tested with different browsers, as the problem, if any, is framework specific.
I'm available to chat if the testing request is not detailed enough. Thank you!

**Comment by sflorean** [comment]
Tested on Chrome, following the same steps, and with TalkBack enabled, and I got the same results, I am not able to interact with 1 tap on PiP, only with 2.  I will close this issue.

**Comment by mcarare** [comment]
Closing as per https://github.com/mozilla-mobile/fenix/issues/10389#issuecomment-663429727.
